app/konnect/runtime-manager/composite-runtime-groups/migrate.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Migrate configuration into composite runtime groups
+content_type: how-to
+badge: enterprise
+---
+
+There are a few situations in which you might want to migrate existing configuration into a composite runtime group.
+
+* Migrate standard runtime group configuration into a composite group. In this scenario, you are migrating config from Konnect to a different group in Konnect.
+* Migrate self-managed Kong Gateway workspace configuration into a composite group in Konnect. In this scenario, you are migrating from self-managed to cloud managed.
+
+A composite runtime group **can't be configured directly**. It compiles configuration from its child runtime groups.
+
+Therefore, when migrating, you will need at least two new groups: a composite runtime group and a standard runtime group.
+
+## Prerequisites
+
+* **Runtime group admin** permissions
+* decK v1.12 or later [installed](/deck/latest/installation/)
+* You have a Konnect access token and you have [made sure that decK can connect to your account](/konnect/runtime-manager/declarative-config/)
+
+## Prepare runtime groups for migration
+
+1. Create a composite runtime group:
+
+    ```sh
+    curl -i -X POST https://<region>.api.konghq.com/v2/runtime-groups \
+        -H "Authorization: Bearer <your_KPAT>" \
+        --data name=CRG \
+        --data cluster_type=CLUSTER_TYPE_COMPOSITE
+    ```
+
+1. Create a new standard runtime group:
+
+    ```sh
+    curl -i -X POST https://<region>.api.konghq.com/v2/runtime-groups \
+        -H "Authorization: Bearer <your_KPAT>" \
+        --data name=SRG1 \
+        --data cluster_type=CLUSTER_TYPE_HYBRID
+    ```
+
+1. Add the new group to the composite runtime group as a member:
+
+    ```sh
+    curl -i -X POST https://<region>.api.konghq.com/v2/runtime-groups/<composite-group-ID>/composite-memberships/add \
+        -H "Authorization: Bearer <your_KPAT>" \
+        --json '{"members": [{"id": "062e2f2c-0f42-4938-91b4-f73f399260f5"}]}'
+    ```
+
+    When adding a standard group to a composite, make sure it has no connected runtime instances.
+
+    A composite runtime group can contain up to 256 runtime groups. 
+    You can add or remove up to 50 child runtime groups at a time.
+
+    Each standard group can have up to 5 parent composite runtime groups.
```

Comment:
nitpick: This is duplicated documentation that also exists on other pages. Do we want to repeat information? Also, use of "child/parent"?

Reply:
Will remove this one, since it's not really relevant when you're adding one runtime group.